src/style.css

```
@@ -1,3 +1,72 @@
+html {
+  --main-bg-color: #00acdc;
+}
```

variables should be in `:root`
```suggestion
:root {
  --main-bg-color: #00acdc;
}
```